### PR TITLE
Yuchen/clo typical ensembles

### PR DESCRIPTION
**Pull request by YuchenMu**
clo_typical_ensembles done

### CODE REVIEW

File: src/utilities/utilities.js [hunk 1]
```diff
@@ -749,3 +749,41 @@ export const met_typical_tasks = {
...
   Wrestling: 7.8,
 };
+
+/**
+ * Total Clothing insulation of typical ensembles
+ * @public
+ * @memberof reference_values
+ * @docname Typical ensembles insulation, [clo]
+ *
+ * @param {string} ensembles - Typical ensembles
```
Comment (szaldivar) [inline src/utilities/utilities.js:759]: can you specify the strings that the function can take so users get autocomplete:
```javascript
{"Walking shorts, short-sleeve shirt" | "Typical summer indoor clothing"...}
```

**Review by szaldivar — CHANGES_REQUESTED** [review]

File: src/utilities/utilities.js (hunk 1 above)
Comment (YuchenMu) [reply src/utilities/utilities.js]: I'm not sure if I got it properly. I updated the @param part and listed all the valid ensembles, not sure if that's what you expected.

Comment (szaldivar) [reply src/utilities/utilities.js]: yes, but you need to remove the `ensembles :` part

File: src/utilities/utilities.js [hunk 2]
```diff
@@ -749,3 +749,53 @@ export const met_typical_tasks = {
...
   Wrestling: 7.8,
 };
+
+/**
+ * Total Clothing insulation of typical ensembles
+ * @public
+ * @memberof reference_values
+ * @docname Typical ensembles insulation, [clo]
+ *
+ * @param {{ensembles : "Walking shorts, short-sleeve shirt" | "Typical summer indoor clothing" |
```
Comment (szaldivar) [inline src/utilities/utilities.js:759]:
```suggestion
 * @param {"Walking shorts, short-sleeve shirt" | "Typical summer indoor clothing" |
```

File: src/utilities/utilities.js [hunk 3]
```diff
@@ -749,3 +749,53 @@ export const met_typical_tasks = {
...
+ * "Knee-length skirt, short-sleeve shirt, sandals, underwear" | "Trousers, long-sleeve shirt" |
+ * "Knee-length skirt, long-sleeve shirt, full slip" | "Sweat pants, long-sleeve sweatshirt" |
+ * "Jacket, Trousers, long-sleeve shirt" | "Typical winter indoor clothing"}} - Typical ensembles. One of:
```
Comment (szaldivar) [inline src/utilities/utilities.js:762]:
```suggestion
 * "Jacket, Trousers, long-sleeve shirt" | "Typical winter indoor clothing"} ensembles - Typical ensembles. One of:
```

**Review by szaldivar — CHANGES_REQUESTED** [review]

**Review by szaldivar — APPROVED** [review]

**Pull request merged**